### PR TITLE
documentation/ZENKOIO-70_server-side-encryption

### PR DESCRIPTION
**Pull request by wabernat**
This PR: expands, clarifies the description of server-side encryption for location setup.

It fixes ZENKOIO-70

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e approve

**Review by jonathan-gramain — APPROVED** [review]

File: docs/docsource/operation/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst [hunk 1]
```diff
@@ -156,14 +156,25 @@ the target cloud.
...
-include``"x-amz-server-side-encryption": "AES256"`` in API calls to AWS.
+Encryption-based transfer protocols ensure your credentials and transmitted
+information are secure while in transit. The S3 API also offers encryption and
+key management services to protect information stored on cloud drives. From
+Orbit, clicking **Server Side Encryption** when setting up a location creates a
+location with encryption enabled for all objects stored there. Encryption is set
+at the bucket level, not at the object level. Object encryption is delegated to
+the cloud storage system.
+
+Server-side encryption is based on the x-amz-server-side-encryption API
```
Comment (dora-korpar) [inline docs/docsource/operation/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst]: I believe `x-amz-server-side-encryption` is a header, not an API call

Comment (wabernat) [reply docs/docsource/operation/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst]: fixed

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e reset

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e reset

**Pull request merged**